src/colorizer/utils/react_utils.ts

```diff
@@ -510,6 +515,17 @@ export const useAnnotations = (): AnnotationState => {
     setIsAnnotationEnabled(false);
   };
 
+  const replaceAnnotationData = (newAnnotationData: AnnotationData): void => {
+    annotationDataRef.current = newAnnotationData;
+    if (newAnnotationData.getLabels().length > 0) {
```

Comment:
Do we care about an else statement for newAnnotationData.getLabels().length === 0 here?